docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,162 @@
+
+## List of System Properties
+
+#### paper.playerconnection.keepalive:
+
+- **default**: `30`
+- **description**: Controls how long the player connection will wait before closing in seconds.
+
+#### paper.ticklist-excessive-delay-threshold:
+
+- **default**: `60`
+- **description**: Sets the excessive tick delay to the specified number of seconds.
+
+#### timings.bypassMax:
+
+- **default**: `unset`
+- **description**: Allows for bypassing the max number of frames to send to the Aikar's Timings API. Setting this will not permit bypassing the limit unless the API is configured to allow it.
+
+#### LetMeReload:
+
+- **default**: `unset`
+- **description**: Overrides the need to send confirmation message when using the `/reload` command.
+
+#### paper.disableChannelLimit:
+
+- **default**: `unset`
+- **description**: Disables the channel limit for the server.
+
+#### net.kyori.adventure.text.warnWhenLegacyFormattingDetected:
+
+- **default**: `true`
+- **description**: Enables or disables the warning when legacy formatting is detected.
+
+#### Paper.DisableClassPrioritization:
+
+- **default**: `unset`
+- **description**: Disables the class prioritization system - mostly an issue when failing to relocate or shade properly.
+
+#### Paper.disableFlushConsolidate:
+
+- **default**: `unset`
+- **description**: Disables the netty flush consolidation system.
+
+#### Paper.debugDynamicMissingKeys:
+
+- **default**: `unset`
+- **description**: Enables debug logging for missing keys in NBT objects.
+
+#### disable.watchdog:
+
+- **default**: `unset`
+- **description**: Disables the watchdog warning system.
+
+#### paper.explicit-flush:
+
+- **default**: `unset`
+- **description**: Enables explicit flushing of the network channel.
+
+#### Paper.enable-sync-chunk-writes:
+
+- **default**: `unset`
+- **description**: Syncs writes on each write call. This has a performance impact, particularly on hard drives.
+
+#### paper.debug-sync-loads:
+
+- **default**: `unset`
+- **description**: Enables debug logging for sync chunk loads.
+
+#### Paper.ignoreWorldDataVersion:
+
+- **default**: `unset`
+- **description**: Ignores the world data version when loading a world. This is not recommended and will likely cause issues.
+
+#### Paper.allowAsyncChunksSingleCore:
+
+- **default**: `unset`
+- **description**: Allows async chunk loading on single core systems.
+
+#### debug.entities:
+
+- **default**: `unset`
+- **description**: Enables debug log messages for entity information.
+
+#### Paper.bypassHostCheck:
+
+- **default**: `unset`
+- **description**: Bypasses the host pattern matching attempt for the client when connecting to the server.
+
+#### paper.ticklist-warn-on-excessive-delay:
+
+- **default**: `unset`
+- **description**: Enables the warning when a tick list is scheduled with an excessive delay.
+
+#### debug.rewriteForIde:
+
+- **default**: `unset`
+- **description**: Removes the NMS revision from the stack trace to allow for easier debugging in IDEs.
```

Comment:
this isn't only the stack trace, right? this is way more, this did rewrite all plugin calls or something iirc?

Reply:
I'm not sure honestly, I'd need someone else's input here. It seems to be used in the Commodore class in CB

Reply:
Yes it should rewrite plugins calls from
like `org/bukkit/craftbukkit/1_19_R1/....`  to `org/bukkit/craftbukkit/....`

```java
getAndRemove.put( "org/bukkit/".concat( "craftbukkit/" + NMS_REVISION_PACKAGE ), NMS_REVISION_PACKAGE );
```